draftjs_exporter/entity_state.py

```diff
@@ -39,28 +40,22 @@ def get_entity_details(self, entity_key):
 
         return details
 
-    def get_entity_decorator(self, type_):
-        if type_ not in self.entity_decorators:
-            raise EntityException('Decorator "%s" does not exist in entity_decorators' % type_)
-
-        decorator = self.entity_decorators[type_]
-
-        return decorator
-
-    def render_entitities(self, style_node):
+    def render_entities(self, style_node):
 
         if self.completed_entity is not None:
-            # self.element_stack.append(style_node)
             entity_details = self.get_entity_details(self.completed_entity)
-            decorator = self.get_entity_decorator(entity_details['type'])
+            opts = Options.for_entity(self.entity_decorators, entity_details['type'])
```

Comment:
Wasn't strictly required, but I thought it would be nicer to align the config handling of all types of "components".